### PR TITLE
Feature/Show WILD price in Ui (Buy Token Redirect Flow)

### PR DESCRIPTION
**Pull request by domw30**
Link to associated zer0 card:  [Show WILD price in UI](https://zer0.io/a/network/tasks/board/e85a99cf-665f-427c-bb0b-2b4e12ba207a/20b83153-e73a-4aad-bd5e-779edd6b2fe8)

-----

## 1. Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] zer0 card has been moved to the Code Review column
- [x] zer0 card has a link to this PR
- [x] A reviewer has been tagged in the zer0 card

## 2. PR type
- [x] Feature

## 3. What is the current behaviour?
- The price ui element and dropdown behaviour does not currently exist in the Title Bar.
- The future button on the title bar currently has conditionals for the text 'Mint'.

## 4. What is the new behaviour?
- Price element added with dropdown behaviour. Dropdown content includes external url links to 'Buy' wild or 'View' wild.
- Future button will only display text 'Mint' to shorten.

## 5. Other information
- [x] ~~**Action Required** : The percentage displayed in the UI is currently a placeholder. I wasn't sure if we are able to pull this data from the BE or if a certain API needs to be called for this. - @colbr are you able to provide direction on this, thanks.~~
- [x] ~~**Action Required** : This will require a design review as well. The anchor tags and hover state background colours differ slightly in Figma designs and handover loom video.~~ 
- [x] ~~**Action Required** : @colbr are the images okay size wise? I exported them off Figma but seem large.~~

### CODE REVIEW

**Comment by colbr** [comment]
Hey @domw30 - sorry I haven't been checking review requests, @padala is the currently the person for Marketplace reviews.

Looking good to me so far.

**Re: percentage change**
I believe Coingecko gives you this information through their API. This is the API we're currently calling to get the price. I imagine you'll just need to find where we're currently fetching price, and also stash the percent-change data from the API response.

**Re: image sizes**
Sometimes sizing on Figma exports looks a bit odd because the asset may have been rendered with or without some padding around the Frame. I would open the images in inspector and see the exact size of them, then match them to the exact size in Figma. If they're the same but look different, chuck Figma and Chrome side-by-side and adjust in inspector until it looks right

Few other things:
- Can we remove the grow animation when hovering over the WildPriceContainer? It makes the UI look slightly broken if the WildPriceContainer background animates but the tooltip doesn't.
- Try to use SASS conventions where possible in your `.scss` files - it will keep clutter down. i.e. instead of 
```
.WildPriceContainer {
	...
}

.WildPriceContainer::after {
	...
}
```
Do
```
.WildPriceContainer {
	...

        &::after {
                ...
        }
}
```
etc.

That's my feedback for now, tag @padala as a reviewer when you're finished with the last 2 things 😊

**Comment by padala** [comment]
@domw30  Let me know once changes from @colbr review are incorporated. Thanks

**Comment by domw30** [comment]
@padala, updated this branch with the decimal place format. Now ready for review. Thanks!

I haven't removed the grow on hover as suggested. Removing the grow breaks the flow of the designs - if essential I can remove it now or later date.

**Comment by padala** [comment]
@domw30  I'll review it today.

File: src/containers/flows/BuyTokenRedirect/BuyTokenRedirect.module.scss [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+	flex-direction: column;
+	justify-content: center;
+	position: relative;
+	background: none;
+	height: 100%;
+	padding: 4px 16px;
+	cursor: pointer;
+	transition: all 0.2s ease-in-out 0s;
+
+	&::after {
```
Comment (colbr) [inline src/containers/flows/BuyTokenRedirect/BuyTokenRedirect.module.scss:12]: I noticed that the `::after` gradient overlay is above the text which changes the white colour, can we set the z-index so the gradient is behind the text and doesn't change the white point?

(colour values below are the main text colours the user sees in current version - both should be #fff)

Screen Shot 2022-01-14 at 4 59 32 PM

**Review by colbr — CHANGES_REQUESTED** [review]
Looking great @domw30, 2 small changes and it's good to merge! 🔥

Apologies if my previous feedback was a little ambiguous re: grow animation

**Comment by domw30** [comment]
@colbr mate, updated with the suggestions above. Thanks man!

**Review by colbr — APPROVED** [review]
Nice work @domw30 , looks perfect

**Pull request merged**